components/widgets/admin_console/admin_header.jsx

```diff
@@ -4,14 +4,25 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
-const AdminHeader = (props) => (
-    <h3 className='admin-console-header'>
-        {props.children}
-    </h3>
-);
+export default class SettingsGroup extends React.Component {
+    static get propTypes() {
+        return {
+            children: PropTypes.node.isRequired,
+            button: PropTypes.bool,
+        };
+    }
 
-AdminHeader.propTypes = {
-    children: PropTypes.node.isRequired,
-};
+    render() {
+        let headerClass = '';
 
-export default AdminHeader;
+        if (this.props.button) {
+            headerClass = 'justify-content-between';
```

Comment:
Instead of adding a new prop for this, couldn't we just add `justify-content: space-between;` to the `admin-console__header` class? I think it does nothing when there's only one child, so I can't think of any cases where we wouldn't want it.

Reply:
Yup we can do that.